models/calendar.yml

```
@@ -414,15 +414,14 @@
 
     - 9:00 PM:
         type: Talk
-        emoji: 🔍
+        emoji: 🐈
         title: Job hunting for happiness
         description: |
-          A framework for job hunting to maximize happiness. Essentially defining and prioritizing needs/wants, figuring out how to ask quantitative questions to get qualitative answers, and suggestions for follow ups.<br><br>
-          I make very deliberate decisions when job hunting and have used this process to great success for the last 6 or so years. I want to share with other people so that they can be happy with new jobs as well!
+          A framework for job hunting to maximize happiness. How to define and prioritize needs/wants, figuring out how to ask quantitative questions to get qualitative answers, and suggestions for follow ups.<br><br>
```

```suggestion
          A framework for job hunting to maximize happiness. How to define and prioritize needs/wants, figuring out how to ask quantitative questions to get qualitative answers, and suggestions for follow ups.
```

I'd remove the `<br><br>` since the second paragraph has been removed!